WS2012R2/lisa/remote-scripts/ica/UIO_Basic.sh

```diff
@@ -0,0 +1,212 @@
+#!/bin/bash
+
+########################################################################
+#
+# Linux on Hyper-V and Azure Test Code, ver. 1.0.0
+# Copyright (c) Microsoft Corporation
+#
+# All rights reserved.
+# Licensed under the Apache License, Version 2.0 (the ""License"");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# THIS CODE IS PROVIDED *AS IS* BASIS, WITHOUT WARRANTIES OR CONDITIONS
+# OF ANY KIND, EITHER EXPRESS OR IMPLIED, INCLUDING WITHOUT LIMITATION
+# ANY IMPLIED WARRANTIES OR CONDITIONS OF TITLE, FITNESS FOR A PARTICULAR
+# PURPOSE, MERCHANTABLITY OR NON-INFRINGEMENT.
+#
+# See the Apache Version 2.0 License for specific language governing
+# permissions and limitations under the License.
+#
+########################################################################
+
+# Convert eol
+dos2unix utils.sh
+
+# Source utils.sh
+. utils.sh || {
+	echo "Error: unable to source utils.sh!"
+	echo "TestAborted" > state.txt
+	exit 2
+}
+
+#######################################################################
+# Check kernel version is newer than the specified version
+# if return 0, the current kernel version is newer than specified version
+# else, the current kernel version is older than specified version
+#######################################################################
+CheckVMFeatureSupportStatus()
```

Comment:
can this be moved to utils.sh and be sourced from there?
This function already exists on the PS side in tcutils.ps1, so it might be more useful for future uses in the utils functions.
Thank you!